src/client.ts

```diff
@@ -132,7 +132,7 @@ export function useGrid({
  */
 export const ClientContext = createContext<Client>(
 	new Client({
-		base: new URL(window.location.href),
+		base: new URL(globalThis?.window?.location.href || 'http://localhost'),
```

Comment:
I think `globalThis` maps to window in a browser environment therefore this would probably do...

 ```
globalThis?.location.href || 'http://localhost'

Reply:
That is correct :) thanks.